concepticondata/conceptlists/Greenhill-2023-121.tsv

```diff
@@ -0,0 +1,122 @@
+ID	NUMBER	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Greenhill-2023-121-1	1	no, not	2279	NO OR NOT
+Greenhill-2023-121-2	2	many	1198	MANY
+Greenhill-2023-121-3	3	one	1493	ONE
+Greenhill-2023-121-4	4	two	1498	TWO
+Greenhill-2023-121-5	5	big	1202	BIG
+Greenhill-2023-121-6	6	old	1229	OLD
+Greenhill-2023-121-7	7	small	1246	SMALL
+Greenhill-2023-121-8	8	short	1645	SHORT
+Greenhill-2023-121-9	9	long	1203	LONG
+Greenhill-2023-121-10	10	new	1231	NEW
+Greenhill-2023-121-11	11	good	1035	GOOD
+Greenhill-2023-121-12	12	round	1395	ROUND
+Greenhill-2023-121-13	13	heavy	1210	HEAVY
+Greenhill-2023-121-14	14	dry	1398	DRY
+Greenhill-2023-121-15	15	sharp (blade)	1396	SHARP
+Greenhill-2023-121-16	16	dirty	1230	DIRTY
+Greenhill-2023-121-17	17	full	1429	FULL
+Greenhill-2023-121-18	18	cold	1287	COLD
+Greenhill-2023-121-19	19	hot	1286	HOT
+Greenhill-2023-121-20	20	woman	962	WOMAN
+Greenhill-2023-121-21	21	man	1554	MAN
+Greenhill-2023-121-22	22	person	683	PERSON
+Greenhill-2023-121-23	23	fish	227	FISH
+Greenhill-2023-121-24	24	bird	937	BIRD
+Greenhill-2023-121-25	25	dog	2009	DOG
+Greenhill-2023-121-26	26	louse	1392	LOUSE
+Greenhill-2023-121-27	27	snake	730	SNAKE
+Greenhill-2023-121-28	28	tree	906	TREE
+Greenhill-2023-121-29	29	bark	1204	BARK
+Greenhill-2023-121-30	30	leaf	628	LEAF
+Greenhill-2023-121-31	31	root	670	ROOT
+Greenhill-2023-121-32	32	seed	714	SEED
+Greenhill-2023-121-33	33	blood	946	BLOOD
+Greenhill-2023-121-34	34	meat	634	MEAT
+Greenhill-2023-121-35	35	skin	763	SKIN
+Greenhill-2023-121-36	36	bone	1394	BONE
+Greenhill-2023-121-37	37	grease, fat	323	FAT (ORGANIC SUBSTANCE)
+Greenhill-2023-121-38	38	egg	744	EGG
+Greenhill-2023-121-39	39	horn	1393	HORN (ANATOMY)
+Greenhill-2023-121-40	40	tail	1220	TAIL
+Greenhill-2023-121-41	41	feather	1201	FEATHER
+Greenhill-2023-121-42	42	hair	1040	HAIR
+Greenhill-2023-121-43	43	head	1256	HEAD
+Greenhill-2023-121-44	44	ear	1247	EAR
+Greenhill-2023-121-45	45	eye	1248	EYE
+Greenhill-2023-121-46	46	nose	1221	NOSE
+Greenhill-2023-121-47	47	mouth	674	MOUTH
+Greenhill-2023-121-48	48	tooth	1380	TOOTH
+Greenhill-2023-121-49	49	tongue	1205	TONGUE
+Greenhill-2023-121-50	50	claw, nail	72	CLAW
+Greenhill-2023-121-51	51	foot	1301	FOOT
+Greenhill-2023-121-52	52	knee	1371	KNEE
+Greenhill-2023-121-53	53	hand	1277	HAND
+Greenhill-2023-121-54	54	arm	1673	ARM
+Greenhill-2023-121-55	55	rope	1218	ROPE
+Greenhill-2023-121-56	56	belly	1251	BELLY
+Greenhill-2023-121-57	57	navel, bellybutton	1838	NAVEL
+Greenhill-2023-121-58	58	neck	1333	NECK
+Greenhill-2023-121-59	59	breast	1402	BREAST
+Greenhill-2023-121-60	60	heart	1223	HEART
+Greenhill-2023-121-61	61	liver	1224	LIVER
+Greenhill-2023-121-62	62	drink	1401	DRINK
+Greenhill-2023-121-63	63	eat	1336	EAT
+Greenhill-2023-121-64	64	bite	1403	BITE
+Greenhill-2023-121-65	65	vomit	1278	VOMIT
+Greenhill-2023-121-66	66	see	1409	SEE
+Greenhill-2023-121-67	67	hear	1408	HEAR
+Greenhill-2023-121-68	68	know	1410	KNOW (SOMETHING)
```

Comment:
KNOW?

Reply:
Here, one could also check the individual data points, to see if it is rather "know (something)". The "prototypcial" KNOW is always "KNOW (SOMETHING)", however.

Reply:
I checked and it seems that the best mapping for this is `KNOW (SOMETHING)` indeed. I'll, therefore, leave it as it is.